Sources/UIDeviceExtensions.swift

```diff
@@ -85,6 +85,18 @@ extension UIDevice {
         return UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
     }
 
+    /// EZSE: Returns path of documents directory
+    class func documentsDirectoryPath() -> String {
```

Comment:
@Khalian I agree, where do you thing would be the best place to put it? Maybe NSFileManager?